tests/Helpers/Factory.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Helper\Tests\Helpers;
+
+use Algolia\AlgoliaSearch\Helper\SearchClient;
+
+final class Factory
+{
+    private static $instance;
+
+    private static $lang = 'php-helper';
+
+    private $client;
+
+    /**
+     * Returns a new index name based on the given `name`.
+     *
+     * @param string $name
+     *
+     * @return string
+     */
+    public static function getIndexName($name)
+    {
+        if (!self::$instance) {
+            self::$instance = getenv('TRAVIS') ? getenv('TRAVIS_JOB_NUMBER') : get_current_user();
+        }
+
+        self::$lang = getenv('TRAVIS') ? 'TRAVIS_'.self::$lang : self::$lang;
+        if (getenv('COMPOSER_FLAGS') === '--prefer-lowest') {
+            self::$lang .= '_lowest';
```

Comment:
```suggestion
            self::$lang .= 'lowest';
```

Reply:
For the moment I will keep '_lowest' until we don't have hand on the api key dealer, this is the string we need to match : https://github.com/algolia/api-key-dealer/blob/c6a21aae889512edbb20082f701dd2a5d7b7ae4b/config/repositories.php#L25